docs/dev_instructions.rst

```diff
@@ -1,23 +0,0 @@
-.. _dev-instructions:
-
-=======================
-Developers Instructions
-=======================
-
-.. contents::
-   :local:
-
-In this guide we present the developers instructions for the Unified-Planning library.
-
-Code formatting
-===============
-
-We decided to use `black <https://black.readthedocs.io>`_ to automatically check and format
-the code according to a predefined code style.
-
-For maintaining code formatting and styling, instead of humans correcting the linting mistakes,
-we use `pre-commit <https://pre-commit.com/>`_. After installing it, execute `pre-commit install`:
-this will install git hooks in .git/ directory of the project.
-Then, when you commit staged Python files, before committing pre-commit hooks are executed and
-black checks the code formatting. If every check passes, the commit is made else, code
-is automatically formatted and sent back for review.
```

Comment:
I wasn't sure if the developer guidelines would go into documentation or the repository README.md. So I haven't added it as a part of the documentation. 

Will it be okay to add the instructions to the repository since it would be a better fit there?

Reply:
Yes, I think it is fine! I noticed that in the README the notebooks link is broken and needs to be fixed.

Reply:
In the examples.rst file the notebooks links should be fixed before merging.

Reply:
I have fixed the github / colab links for examples.rst and notebooks.